api/formatter.py

```diff
@@ -0,0 +1,7 @@
+def formatter(game):
```

Comment:
This PR is massive. A better way of doing this is by splitting into this multiple PRs and have one feature for branch